.gitignore

```diff
@@ -2,6 +2,10 @@
 #
 .DS_Store
 
+# Do not track local package
+react-native-ui-lib-5.0.0.tgz
```

Comment:
Will it always be 5.0.0?

Reply:
I believe it comes from our `package.json`'s `version`, so it should not change since we do not really update it.